Automattic/sensei
# Fix for filtering existing questions on PHP 7.1.4

When running Sensei on PHP 7.1.4, the following PHP warning is being logged when filtering existing questions for a lesson:

`PHP Warning:  Illegal string offset 'html' in /app/public/wp-content/plugins/sensei/includes/class-sensei-lesson.php on line 1381`

This fixes #1753 by initializing the `$return` variable to an array rather than a string.

## Testing

1. Ensure you're running PHP 7.1.4.
2. Add more than 10 questions via _Questions_ > _Add New_.
3. Edit a lesson and click on the _Existing Questions_ tab.
4. Filter the questions. Only those questions containing that filter keyword should be displayed.
5. Click _Next_. The next page of questions should be shown.
